python/kvikio/kvikio/defaults.py

```diff
@@ -181,11 +181,9 @@ def is_compat_mode_preferred() -> bool:
     return kvikio._lib.defaults.is_compat_mode_preferred()
 
 
-@kvikio_deprecation_notice('Use kvikio.defaults.get("compat_mode") instead')
+@kvikio_deprecation_notice("Use :meth:`kvikio.defaults.get()` instead.", "25.04")
```

Comment:
Deprecations would take place in 25.06, followed by removal in 25.08 at the earliest.
```suggestion
@kvikio_deprecation_notice("Use :meth:`kvikio.defaults.get()` instead.", "25.06")
```

Reply:
Actually. This part of the code in `defaults.py` has been deprecated since the `25.04` release.